src/templates/bespoke/transition.ts

```diff
@@ -97,7 +97,15 @@ const bespokeTransition = (deck) => {
         })
         .filter((v): v is string => !!v)
     )
-  )
+  ).then(() => {
+    // Workaround for Chrome's animation jank: Disable duration variables in keyframes after prefetch
+    document.querySelectorAll<HTMLStyleElement>('style').forEach((style) => {
+      style.innerHTML = style.innerHTML.replace(
+        /--marp-transition-duration:[^;}]*[;}]/g,
+        (matched) => matched.slice(0, -1) + '!important' + matched.slice(-1)
+      )
+    })
```

Comment:
Chrome seems to bring animation jank if set not interpolatable value to the last keyframe. A custom transition with simple definition will set `--marp-transition-duration` to the last keyframe due to auto-reversed animation.

As a workaround, we will try to sanitize custom properties in `<style>` tag after prefetching keyframes. In `@keyframes`, properties with `!important` will be ignored.